clippy_lints/src/manual_clamp.rs

```diff
@@ -17,6 +18,7 @@ use rustc_middle::ty::Ty;
 use rustc_session::impl_lint_pass;
 use rustc_span::symbol::sym;
 use rustc_span::Span;
+use std::cmp::Ordering;
 use std::ops::Deref;
 
 declare_clippy_lint! {
```

Comment:
Could you add a `### Limitations` section to the lint description, clarifying that this lint only triggers when the min and max values are known at compile time?